clang/lib/AST/ASTImporter.cpp

```diff
@@ -5943,10 +5954,13 @@ ExpectedDecl ASTNodeImporter::VisitClassTemplateDecl(ClassTemplateDecl *D) {
 
         // FIXME: sufficient conditon for 'IgnoreTemplateParmDepth'?
         bool IgnoreTemplateParmDepth =
```

Comment:
Change of this condition was needed to make a failing test `ImportOfRecursiveFriendClassTemplateWithNonTypeParm` to pass.